### ISSUE TITLE
Unhandled exception

### ISSUE THREAD

**Issue by MrSkelegreen** [AvaloniaUtils/AsyncImageLoader.Avalonia#7]
Hi. I receive this unhandled exception:

```
The type initializer for 'AsyncImageLoader.ImageLoader' threw an exception.
 ---> System.IO.FileNotFoundException: Could not load file or assembly 'Avalonia.Visuals, Version=0.10.0.0, Culture=neutral, PublicKeyToken=c8d484a7012f9a8b'.
 ```
I tried to install NP.Avalonia.Visual, but it but it did not help.

**Comment by SKProCH** [comment]
Hello, @MrSkelegreen 

For Avalonia 11 you should use the '3.0.0-avalonia11*' version.

**Comment by MrSkelegreen** [comment]
@SKProCH, yes, the image has appeared. But does it work on mobile devices? It seems that my phone does not display a picture. (I am working on a cross platform solution)

**Comment by SKProCH** [comment]
It should work on all platforms, because it uses a default HttpClient and default Avalonia's Image under the hood.

Make sure what you gave an internet permission to the app (for Android)

**Comment by MrSkelegreen** [comment]
In theory, there is permission, because I use the HttpClient to receive responses from the API.

**Comment by MrSkelegreen** [comment]
![dDHsMiuKABM](https://user-images.githubusercontent.com/104663261/211274157-744779b7-bfd8-4bfa-9e44-f8671f402bbf.jpg)

![vOplPRuusiQ](https://user-images.githubusercontent.com/104663261/211274175-d1f319ef-edde-4872-a473-66c122b4c0ae.jpg)

**Comment by SKProCH** [comment]
Look at log, there should be `Information` message with text `Failed to resolve image from request with uri` (ensure what you have information messages enabled in log). Or you can set breakpoint and trace image downloading in `BaseWebImageLoader` (or implement your own `IAsyncImageLoader` and set in `ImageLoader.AsyncImageLoader` property).

Also, i'll try to look at this in the nearest future.

**Comment by MrSkelegreen** [comment]
Honestly I wanted to load images from URL in my ListBox, and it works for some reason. Thanks for help!
```
<ListBox IsVisible="{Binding }" Grid.Row="4" Grid.Column="1" Grid.ColumnSpan="2" Name="ListBox" 
                 Items="{Binding Recipes}" HorizontalAlignment="Center" VerticalAlignment="Center"
                 AutoScrollToSelectedItem="False">
            <ListBox.ItemTemplate>
                <DataTemplate>
                    <Grid>
                        <Image asyncImageLoader:ImageLoader.Source="{Binding recipe.image}" Width="300" Height="140"/>
                        <TextBlock Text="{Binding recipe.Title}" Width="300" Height="140"/>    
                    </Grid>
                </DataTemplate>
            </ListBox.ItemTemplate>
        </ListBox>
```
![image](https://user-images.githubusercontent.com/104663261/211295967-3e9c0409-de42-4e6c-87da-c87d230bd52d.png)

**Comment by SKProCH** [comment]
So, seems like everything works. Feel free to reopen/open a new issue if problem persists

**Comment by WarreTh** [comment]
I currently cannot display the image on any mobile device. When using desktop, it does work.
And when i open the url link in the browser of the phone, it also works.

```
<Image asyncImageLoader:ImageLoader.Source="{Binding NewestImageUrl}" Stretch="Uniform"/>

```

**Comment by SKProCH** [comment]
Same here:

> It should work on all platforms, because it uses a default HttpClient and default Avalonia's Image under the hood.

> Make sure what you gave an internet permission to the app (for Android)
Look at log, there should be `Information` message with text `Failed to resolve image from request with uri` (ensure what you have information messages enabled in log). Or you can set breakpoint and trace image downloading in `BaseWebImageLoader` (or implement your own `IAsyncImageLoader` and set in `ImageLoader.AsyncImageLoader` property).

**Comment by WarreTh** [comment]
> Look at log

Where should i look at the log? In the debugging console? Sorry im very new to avalonia

**Comment by SKProCH** [comment]
Yes, it should be in Debug output window:

![Image](https://github.com/user-attachments/assets/d796fa4a-8a15-4502-b96b-5af130bdd03a)

**Comment by WarreTh** [comment]
But how can i see debugging info on my android? Using the log viewer?

**Comment by SKProCH** [comment]
You probably can attach (or just launch the app) with debugger from VS or Rider (whatever IDE you use), and after that see logs from IDE.
